README.md

```diff
@@ -0,0 +1,141 @@
+# Rootless Pixel Launcher
+
+![Rootless Pixel Launcher in action](https://i.imgur.com/wTCa3uW.gif)
+
+The official Pixel Launcher has a lot of exclusive features like the Google Now panel and G Search Pill that only work while it is running as a system app. When regular users install the APK file they get a gimped version of the app that refuses to give them these features. However, after seeing the ParanoidAndroid commits to get the Google Now panel working I realized these were not limitations of being installed as a normal app, instead Google intentionally made it work this way, most probably to keep the features exclusive to the Pixel phones as a selling point like HDR+. The older Google Now Launcher could make use of the Google Now panel too when you installed it from the Play Store.
+
+--------
+
+**Downloads**: https://github.com/amirzaidi/Launcher3/releases
+
+**Pictures**: https://goo.gl/photos/C6kWrgMV3jMfL2Ld9
+
+**Sources**: https://github.com/amirzaidi/Launcher3/commits/o
+
+*Report bugs with a logcat, if you don't know how to take one please use Google. I do not ask for donations and I did not add ads in the launcher. If you want to support me, wish me a happy birthday on September the 6th.* 
+
+--------
+
+## Version 2.1 is out!
```

Comment:
Unnecessary -> can easily be seen in releases section

Reply:
I wanted to leave it so that people can check from here what is the latest version. It's fine for me to remove it.

Reply:
The thing is that you have to update the readme with every version then. If @amirzaidi is ok with it thats fine.

Reply:
You're right. Let's wait for his response.